.gitignore

```diff
@@ -8,3 +8,8 @@
 /spec/reports/
 /tmp/
 .overcommit_gems.rb.lock
+.tool-versions
```

Comment:
The same I said in `devise-jwt` PR :smiley: Could you please track them locally in your `.git/info/exclude` instead?

Reply:
done